app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -515,14 +515,14 @@ def resolve_ip_address(address)
     # @return [Exception] The exception that the ManageIQ expects.
     #
     # @api private
-    #
+    # rhevm_metrics_connect_options
     def adapt_metrics_error(error)
       case error
-      when PGError
+      when PG::Error
         message = error.message
         message = error.message[6..-1] if message.starts_with?('FATAL:')
         message = message.strip
-        _log.warn("PGError: #{message}")
+        _log.warn("PG::Error: #{message}")
```

Comment:
perhaps replace "PG::Error" with "#{error.class.name}" ?

Reply:
Perfect, I'll fix it